docs/api/v1.21.md

```diff
@@ -2836,7 +2836,12 @@ Content-Type: application/json
 **JSON parameters**:
 
 - **Name** - The new network's name. this is a mandatory field
-- **CheckDuplicate** - Requests daemon to check for networks with same name. Defaults to `false`
+- **CheckDuplicate** - Requests daemon to check for networks with same name. Defaults to `false`.
+    Since Network is primarily keyed based on a random ID and not on the name, 
+    and network name is strictly a user-friendly alias to the network which is uniquely identified using ID, 
+    there is no guaranteed way to check for duplicates across a cluster of docker hosts. 
```

Comment:
@allencloud Actually it is not only specific to the multi host scenario. Even for concurrent bridge networks creation, this may happen, for the same reason the networks object in the local store are keyed by ID.